<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xfwnd_debug</class>
 <widget class="QDialog" name="xfwnd_debug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Memory</string>
       </attribute>
       <widget class="QPushButton" name="pb_KillAll">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;&gt;&gt; Kill All &lt;&lt;&lt;</string>
        </property>
       </widget>
       <widget class="QListWidget" name="l_MemUnits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>331</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ulist">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ulist</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Sound System</string>
       </attribute>
       <widget class="QPushButton" name="pb_SndStart">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Sound System</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <widget class="QPushButton" name="pb_InitCfg">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Video</string>
       </attribute>
       <widget class="QPushButton" name="pb_InitVideo">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Bus</string>
       </attribute>
       <widget class="QPushButton" name="pb_BusStats">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Statistics</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_Channel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>CHANNEL_BUS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHANNEL_31</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="le_ID">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Channel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Data ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_Data">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_Broadcast">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Broadcast</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>xfwnd_debug</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>xfwnd_debug</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
